[DOCS] Add query rules UI links to reference content (#133219)
elastic/elasticsearch: docs/reference/elasticsearch/rest-apis/searching-with-query-rules.md

* A user-entered query string
* Personalized metadata about users (e.g. country, language, etc)
* A particular topic

> * A referring site and so on

Reviewer: ```suggestion
* A referring site
```

personal opinion: I don't think we need either, because "For example" makes it clear this isn't an exhaustive list

* Personalized metadata about users (e.g. country, language, etc)
* A particular topic
* A referring site

Query rules define a metadata key that will be used to match the metadata provided in the [rule retriever](/reference/elasticsearch/rest-apis/retrievers/rule-retriever.md) with the criteria specified in the rule.

When a query rule matches the rule metadata according to its defined criteria, the query rule action is applied to the underlying `organic` query.

For example, a query rule could be defined to match a user-entered query string of `pugs` and a country `us` and promote adoptable shelter dogs if the rule query met both criteria.

You can create and manage query rules using either:
- [Query rules API]({{es-apis}}v9/group/endpoint-query_rules)

> - [Query Rules UI](docs-content://solutions/search/query-rules-ui)

Reviewer: ```suggestion
- [Query Rules UI](docs-content://solutions/search/query-rules-ui.md)
```

> Rules are defined using the [query rules API](https://www.elastic.co/docs/api/doc/elasticsearch/group/endpoint-query_rules) and searched using the [rule retriever](/reference/elasticsearch/rest-apis/retrievers/rule-retriever.md) or the [rule query](/reference/query-languages/query-dsl/query-dsl-rule-query.md).
> 
> Alternatively, rules can be managed through the [Query Rules UI](https://www.elastic.co/docs/solutions/search/query-rules-ui).

Reviewer: ```suggestion
Query rules can be created and managed using either:
- [Query rules API]({{es-apis}}v9/group/endpoint-query_rules)
- [Query Rules UI](docs-content://solutions/search/query-rules-ui)

You can search with query rules using either:
- [Rule retriever](/reference/elasticsearch/rest-apis/retrievers/rule-retriever.md)
- [Rule query](/reference/query-languages/query-dsl/query-dsl-rule-query.md)
```

Note the use of:
- variable for API docs URL
- cross-repo syntax for `docs-content` link

Reviewer: Because there's 2 options each for managing and querying here, probably easier to read in little lists

You can use the [Get query ruleset](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-query-rules-get-ruleset) call to retrieve the ruleset you just created, the [List query rulesets](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-query-rules-list-rulesets) call to retrieve a summary of all query rulesets, and the [Delete query ruleset](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-query-rules-delete-ruleset) call to delete a query ruleset.

> To execute these requests through the Query Rules UI, refer to [Manage query rules](https://www.elastic.co/docs/solutions/search/query-rules-ui#manage-existing-rules).

Reviewer: ```suggestion
To manage rules using the Query Rules UI, refer to [Manage query rules](https://www.elastic.co/docs/solutions/search/query-rules-ui#manage-existing-rules).
```
